src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final Color[] colorArr = Color.values();
+
+    public String getRandomColor() {
+        int colorIndex = random.nextInt(colorArr.length);
+
+        return colorArr[colorIndex].name().toLowerCase();
```

Comment:
According to the checklist, we should use only English in messages/code. Converting the enum name to lowercase might not be necessary unless it is a specific requirement of the task. If the enum names are in English and are meant to be displayed, they should be fine as is. Consider returning the enum name directly unless the task specifies otherwise.